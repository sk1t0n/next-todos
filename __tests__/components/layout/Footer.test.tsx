import { render, screen } from '@testing-library/react';
import { describe } from 'jest-circus';
import Footer from '../../../components/layout/footer';

describe('Footer', () => {
  it('check the snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('should be the text with copyright', () => {
    render(<Footer />);
    expect(screen.getByText(/copyright/i)).toBeInTheDocument();
  });
});
