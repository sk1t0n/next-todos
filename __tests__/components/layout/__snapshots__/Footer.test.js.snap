// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check the snapshot 1`] = `
<DocumentFragment>
  <footer
    class="ant-layout-footer footer"
  >
    Copyright Â© 2021
  </footer>
</DocumentFragment>
`;
